### PR TITLE
SC-4275 - SC International Legacy Theming

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description
added new theme and adjusted colors for it

## Links to Tickets or other pull requests

- https://github.com/schul-cloud/nuxt-client/pull/1145
- https://ticketsystem.schul-cloud.org/browse/SC-4275

## Screenshots of UI changes
image
image
image

### CODE REVIEW

**Comment by ABialoskorski** [comment]
> Current State:
> Opera Momentaufnahme_2020-07-15_152114_localhost
> Target state:
> Anmerkung 2020-07-15 152224
> The sidebar colors do not meet the target state

solved

File: theme/int/style.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+			.cloud-logo{
+				background: $logoBackgroundTablet;
+			}
+		}
+	}
+	ul {
+		li {
+			a:not(.sc-branding) {
+				&.active {
+					color: $secondaryColor;
```
Comment (dominikbruechner) [inline theme/int/style.scss:45]:
```suggestion
					color: $secondaryColorDark;
```
vgl. https://www.sketch.com/s/4d31147a-c78f-460d-a5ff-54b247a0e5cc/a/KPRa5K

Comment (ABialoskorski) [reply theme/int/style.scss:45]: done ;)

Comment (ABialoskorski) [reply theme/int/style.scss:45]: done ;)

**Comment by ABialoskorski** [comment]
> Can you please add a screenshot of the UI changes? Thanks

added ;)

**Pull request merged**